express.js

```diff
@@ -0,0 +1,10 @@
+const express=require("express");
```

Comment:
Consider using ES6 import syntax instead of require for importing modules. This is a more modern approach and aligns with most current JavaScript practices. [medium]